src/components/ConsoleOutput.jsx

```diff
@@ -23,7 +32,14 @@ export default function ConsoleOutput({entry, isActive}) {
         }
       >
         {chevron}
-        {value}
+        {Object.keys(deserialize(value)).map(
```

Comment:
I think we can probably do this more straightforwardly with `deserialize(value).map()`, or the lodash equivalent? Both should pass the array index as the second argument to the iterator function.